Services/UserService/Validators/ErrorsMessages.resx

```diff
@@ -159,4 +159,7 @@
   <data name="LastNameTooShort" xml:space="preserve">
     <value>Last name is too short</value>
   </data>
+  <data name="OldAndNewPasswordsEqual" xml:space="preserve">
+    <value>Old and new passwords are equal</value>
+  </data>
 </root>
```

Comment:
Add new line

Reply:
I don't understand you(

Reply:
> Add new line

xD
Ok, I will start requesting such changes too ;)

Reply:
this part of code generated automatically